code/game/jobs/job/cargo_service.dm

```diff
@@ -111,6 +111,7 @@ Bartender
 /datum/outfit/job/bartender
 	name = "Bartender"
 
+	glasses = /obj/item/clothing/glasses/sunglasses/reagent
```

Comment:
Yeah. If I'm not mistaken, he used to spawn with sunglasses (or maybe it was just people going into maint and grabbing a guaranteed spawn).